tests/unit-tests/dds/DCPS/JsonValueWriter.cpp

```diff
@@ -12,6 +12,18 @@
 using namespace rapidjson;
 using namespace OpenDDS::DCPS;
 
+class TestWriter : public JsonValueWriter<>
+{
+public:
+  TestWriter () : elements_ (0) {};
```

Comment:
```suggestion
  TestWriter () : elements_ (0) {}
```
There's a few of these.